src/internal/NeoEsp32RmtMethod.h

```diff
@@ -539,6 +539,8 @@ template<typename T_SPEED, typename T_CHANNEL> class NeoEsp32RmtMethodBase
     {
         rmt_config_t config;
 

```

Comment:
Shouldn't this work too?

```c++
rmt_config_t config = {};
```
https://en.cppreference.com/w/cpp/language/zero_initialization

Reply:
It all depends on how rmt_config_t is defined.
the `= {}` means to set the struct to its defaults.  If not defined then zero. It does not mean set it to zero even though in this case it would since rmt_config_t has no defaults set.

I believe `= {}` is the right solution though.  It future proofs the code in case any new fields that require a non-zero default value is added.

Reply:
Done